typed-racket-lib/typed-racket/private/type-contract.rkt

```diff
@@ -769,6 +775,18 @@
           [_ type])]))
     #f))
 
+;; True if the arities `arrs` are what we'd expect from a struct predicate
+(define (any->bool? arrs)
+  (match arrs
+   [(list (arr: (list (Univ:))
+                (Values: (list (Result: (Union:
+                  (or (list (Value: #t) (Value: #f))
```

Comment:
Use `(== -Boolean)`, like I said.

Also, something seems to be wrong with the indentation in your DrRacket. :)